config/prod/bwhite-novartis-cml.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
+# Change the stack name to something unique
+stack_name: bwhite-novartis-cml
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "novartis-cml"
+  # The resource owner
+  OwnerEmail: "brian.white@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  # See https://www.bioconductor.org/help/bioconductor-cloud-ami/
+  AMIId: "ami-08333c98cd3b5ada7"
```

Comment:
hmm, we have not tested AMI from here.  do you have anymore info like which distro and version they are using?  i think we only support ubuntu 18.04(bionic) at this time.

Reply:
I don't have more information on this AMI, nor do I know how to find out. I can tell you I got it from https://www.bioconductor.org/help/bioconductor-cloud-ami/ (under connecting to your AMI using HTTP and RStudio).

I chose it because it has a recent version of bioconductor and it includes Rstudio, which allows me to log into an Rstudio session remotely. Can you suggest an AMI that we do support that provides that? In general, is there a list of AMIs that we support?

Reply:
Tyr this one http://www.louisaslett.com/RStudio_AMI/.

Reply:
Thanks, Tom. Trying that ...

Reply:
FYI.. that image from bioconductor is indeed bionic so it should provision without problems.  if you want to give it a try create another PR and we can give it a go.  Details about that image:

```
❯❯❯ aws ec2 --output table describe-images --image-ids ami-08333c98cd3b5ada7  

------------------------------------------------------------------------------
|                               DescribeImages                               |
+----------------------------------------------------------------------------+
||                                  Images                                  ||
|+---------------------+----------------------------------------------------+|
||  Architecture       |  x86_64                                            ||
||  CreationDate       |  2019-05-13T16:10:44.000Z                          ||
||  Description        |  bioc-3.9-ubuntu-18.04-40GB-20190513               ||
||  EnaSupport         |  True                                              ||
||  Hypervisor         |  xen                                               ||
||  ImageId            |  ami-08333c98cd3b5ada7                             ||
||  ImageLocation      |  555219204010/bioc-3.9-ubuntu-18.04-40GB-20190513  ||
||  ImageType          |  machine                                           ||
||  Name               |  bioc-3.9-ubuntu-18.04-40GB-20190513               ||
||  OwnerId            |  555219204010                                      ||
||  Public             |  True                                              ||
||  RootDeviceName     |  /dev/sda1                                         ||
||  RootDeviceType     |  ebs                                               ||
||  SriovNetSupport    |  simple                                            ||
||  State              |  available                                         ||
||  VirtualizationType |  hvm                                               ||
|+---------------------+----------------------------------------------------+|
|||                           BlockDeviceMappings                          |||
||+------------------------------------+-----------------------------------+||
|||  DeviceName                        |  /dev/sda1                        |||
||+------------------------------------+-----------------------------------+||
||||                                  Ebs                                 ||||
|||+--------------------------------+-------------------------------------+|||
||||  DeleteOnTermination           |  True                               ||||
||||  Encrypted                     |  False                              ||||
||||  SnapshotId                    |  snap-01cebeb95a682de1c             ||||
||||  VolumeSize                    |  40                                 ||||
||||  VolumeType                    |  gp2                                ||||
|||+--------------------------------+-------------------------------------+|||

```